doc/api-extensions.md

```diff
@@ -6,23 +6,23 @@ They are all backward compatible and can be detected by client tools by
 looking at the `api_extensions` field in `GET /1.0/`.
 
 
-## storage\_zfs\_remove\_snapshots
+## `storage_zfs_remove_snapshots`
 A `storage.zfs_remove_snapshots` daemon configuration key was introduced.
 
-It's a boolean that defaults to false and that when set to true instructs LXD
+It's a Boolean that defaults to false and that when set to true instructs LXD
```

Comment:
Another candidate for `bool` as "Boolean" really looks wrong here :)

Reply:
Same for other occurrences of Boolean below. 

Worth noting that the first result on Google for `boolean capitalization` shows:
```
Boolean is pronounced BOOL-ee-an. The word "Boolean" should only be capitalized in reference to Boolean logic or Boolean algebra. When referring to the data type in computer programming, the word "boolean" is properly spelled with a lowercase b. Boolean operator examples.
```

And that matches how I'd handle capitalization of that word.

Reply:
That's not what any of the dictionaries say though. And I follow the rule that if something is clearly defined in a dictionary, that has priority over Google.

And imo it doesn't look wrong at all, because "Boolean" IS correct. ;)

Reply:
I think its one of those things in English where there isn't a hard and fast rule, and it evolves over time based on the usage.

I found this article quite interesting as it gives some more name-derived examples that we wouldn't commonly capitalise:

https://blog.plover.com/lang/Boolean.html